<p>This rule detects usage of annotation @PathParam without specifying value (i.e. parameter name). So it can lead to errors after
    obfuscation</p>
<h2>Noncompliant Code Example</h2>
<pre>
... endpointMethod(@PathParam String type) {...}
</pre>
<h2>Compliant Solution</h2>
<pre>
... endpointMethod(@PathParam(value = "type") String type) {...}
</pre>